provider/datagen/src/cldr/transform/datetime/mod.rs

```diff
@@ -152,12 +153,15 @@ mod test {
         let provider = CommonDateProvider::try_from(&cldr_paths as &dyn CldrPaths)
             .expect("Failed to retrieve provider");
 
+        let mut locale = locale!("cs");
+        locale
+            .extensions
+            .unicode
+            .keywords
+            .set(unicode_ext_key!("ca"), unicode_ext_value!("gregory"));
```

Comment:
nit: I think in tests it's more readable to parse a full locale rather than assemble it piecewise.

Reply:
+1 good call

Reply:
that said, _personally_ i find the full locale syntax to be a bit annoying to read, so i don't _mind_ this too much